rust/uefi/linux-bootloader/src/measure.rs

```diff
@@ -60,3 +67,81 @@ pub fn measure_image(system_table: &SystemTable<Boot>, image: PeInMemory) -> uef
 
     Ok(measurements)
 }
+
+/// Performs all the expected measurements for any list of
+/// companion initrds of any form.
+///
+/// Relies on the passed order of `companions` for measurements in the same PCR.
+/// You are responsible for honoring a stable order.
```

Comment:
"The order of companions matters for the measurement. The caller is responsible for a stable order."

I would suggest making a pass through the code and removing all mentions of "you" from the docstrings.

Reply:
I kind of want to make a remark that `you` appears in your docstrings for the initrd loader ;) -- but sure, done.